numpy/core/fromnumeric.py

```diff
@@ -1144,6 +1144,8 @@ def argmax(a, axis=None, out=None, *, keepdims=np._NoValue):
         in the result as dimensions with size one. With this option,
         the result will broadcast correctly against the array.
 
+        .. versionadded:: 1.22.0
+
```

Comment:
Not directly related to this PR, but I noticed https://github.com/numpy/numpy/pull/19211 didn't add a `versionadded` directive.